google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -525,24 +525,39 @@ public boolean exists(BlobSourceOption... options) {
   }
 
   /**
-   * Fetches current blob's latest information. Returns {@code null} if the blob does not exist.
+   * Fetches the latest blob properties. Returns {@code null} if the blob no longer exists.
    *
-   * <p>Example of getting the blob's latest information, if its generation does not match the
-   * {@link Blob#getGeneration()} value, otherwise a {@link StorageException} is thrown.
+   * <p>{@code options} parameter can contain the preconditions. For example, the user might want to
+   * get the blob properties only if the content has not been updated externally. {@code
+   * StorageException} with the code {@code 412} is thrown if preconditions fail.
+   *
+   * <p>Example of retrieving the blob's latest information only if the content is not updated
+   * externally:
    *
    * <pre>{@code
-   * Blob latestBlob = blob.reload(BlobSourceOption.generationNotMatch());
-   * if (latestBlob == null) {
-   *   // the blob was not found
+   * Blob blob = storage.get(BlobId.of(bucketName, blobName));
+   *
+   * doSomething();
+   *
+   * try {
+   *   blob = blob.reload(Blob.BlobSourceOption.generationMatch());
+   * } catch (StorageException e) {
+   *   if (e.getCode() == 412) {
+   *     // the content was updated externally
+   *   } else {
+   *     throw e;
+   *   }
    * }
    * }</pre>
    *
-   * @param options blob read options
-   * @return a {@code Blob} object with latest information or {@code null} if not found
+   * @param options preconditions to fetch
```

Comment:
preconditions to use, see https://cloud.google.com/storage/docs/json_api/v1/objects/get for more information.

Reply:
<img width="475" alt="reload-jdoc" src="https://user-images.githubusercontent.com/58846611/81848714-3660cf00-955e-11ea-873d-11525b53b03c.png">

Reply:
Thanks @dmitry-fa,

use: `preconditions to use on reload, see https://cloud.google.com/storage/docs/json_api/v1/objects/get for more information` instead.

Reply:
done